src/components/atoms/Menu/Menu.component.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import { Menu as AntMenu, MenuProps as AntMenuProps, MenuItemProps, SubMenuProps } from 'antd'
+import AntMenuItem from 'antd/lib/menu/MenuItem'
+import AntSubMenu from 'antd/lib/menu/SubMenu'
+
+import { PreviewProps } from '@/types/preview'
+import { ComponentProps } from '@/types/component'
+
+export interface MenuProps extends PreviewProps, ComponentProps<AntMenuProps> {}
+
+export const Menu = React.forwardRef<HTMLDivElement, MenuProps>((props, ref) => {
+  return (
+    <div ref={ref} style={{ display: 'inline-block' }}>
+      <AntMenu {...props} prefixCls="jitera-menu" />
+    </div>
+  )
+})
+
+export const SubMenu = React.forwardRef<HTMLUListElement, SubMenuProps>((props, ref) => {
```

Comment:
Please export it separately. Because generator has not support nested import yet